string id
FrenetPoint[] path
# FrenetPoint mean_val
# FrenetPoint max_val